test/storage/http.test.js

```diff
@@ -21,25 +21,99 @@ const fs = require('fs-extra');
 const { download, upload } = require('../../lib/storage/http');
 const Asset = require('../../lib/asset');
 const nock = require('nock');
-
 const http = require('@adobe/httptransfer');
+const proxyquire =  require('proxyquire');
+const rewire = require('rewire');
+
+const rewiredHttp = rewire('../../lib/storage/http');
+const getMaxConcurrent = rewiredHttp.__get__('getMaxConcurrent');
+
 
-const oldDownloadFileHttpTransfer = http.downloadFile;
+const oldDownloadFileHttpTransfer = http.downloadFileConcurrently;
+
+const DEFAULT_MAX_CONCURRENT = 8;
+const DEFAULT_PREFERRED_PART_SIZE = 100 * 1024 * 1024; // Default part size is 10mb
 
 describe('http.js', () => {
 
     beforeEach( () => {
-        mockFs();
         process.env.__OW_ACTION_NAME = 'test_action';
     });
     afterEach( () => {
-        http.downloadFile = oldDownloadFileHttpTransfer;
+        // TODO: switch tests that are using this to proxyquire
+        http.downloadFileConcurrently = oldDownloadFileHttpTransfer;
         nock.cleanAll();
+        // mockFs messes with rewire and proxyquire loading in of modules
+        // we must make sure to restore mockFs before using these modules
         mockFs.restore();
         delete process.env.__OW_ACTION_NAME;
         delete process.env.ASSET_COMPUTE_DISABLE_RETRIES;
     });
 
+    describe('getMaxConcurrent', () => {
+        it("no containerMemorySize available, default concurrency",  async () => {
```

Comment:
Please make test cases a little bit more explicit
```suggestion
        it("no containerMemorySize available, default concurrency should be used",  async () => {
```

Reply:
changed this to what I used in the rest of the test cases: `no containerMemorySize available, use default concurrency`